### DISCUSSION TITLE
[k8s/helm-chart]: kratos-selfservice-ui-node in k8s trying to resolve kratos pod and fails ....

### DISCUSSION THREAD

**Discussion by DeepDiver1975** [ory/kratos#3547] [Q&A]
I am running kratos and kratos-selfservice-ui-node in k8s 1.24

Hitting the login button in selfservice-ui results in an error within kratos-serfservice-ui-node that the pod name of the kratos instance cannot be resolved:

```
Error: getaddrinfo ENOTFOUND idm-kratos-5b5d58b5c9-shw5h
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:72:26)
```

![kratos-error-pod-dns](https://github.com/ory/.github/assets/1005065/a484a2e4-b5d9-47e4-bdd3-f0b45da9d011)

**Comment by vinckr** [comment]
Hello @DeepDiver1975 

Hard to tell what the issue is just from the request. 
The error message you're seeing typically indicates a DNS resolution issue. It seems like your application is unable to resolve the hostname of your Ory Kratos instance.
You might want to check your Kubernetes DNS configuration and ensure that your services are correctly set up and discoverable. Also, ensure that your Ory Kratos and kratos-selfservice-ui-node are correctly configured to communicate with each other.
